docs/misc/dvd-remux.md

```diff
@@ -0,0 +1,670 @@
+        |----------------------|----------------------------------------|-------------|--------------------|
+        | 4:3                  | 4320:4739                              | 710.85x486  |                    |
+        |                      | 640:711                                | 711x480     | 711x533            |
+        |                      | 160:177                                | 708x480     |                    |
+        |                      | 10:11                                  | 704x480     | 704x528            |
+        | 16:9                 | 2560:2133                              | 711x480     | 853x480            |
+        |                      | 640:531                                | 708x480     |                    |
+        |                      | 40:33                                  | 704x480     |                    |
+
+    === "PAL"
+
+        | Display Aspect Ratio | Sample Aspect Ratio/Pixel Aspect Ratio | Active Area | Display Resolution |
+        |----------------------|----------------------------------------|-------------|--------------------|
+        | 4:3                  | 128:117                                | 702x576     |                    |
+        |                      | 1132:1035                              | 690x566     |                    |
+
+!!! warning "NTSC to PAL conversions"
+    When working with DVDs that have been converted between NTSC and PAL formats,
+    the standard aspect ratio values may not apply.
+
+    For these cases:
+
+    1. First determine and correct the SAR values of the original format
+    2. Analyze how the conversion process transformed the video
+    3. Adjust your settings to match the original content's intended display
+
+<!-- TODO: Maybe mention dvdunauthor? I only see Linux builds for this, but it may be useful for troubleshooting? -->
+
+### Determining the correct SAR values
+
+The following methods can be used
+to help determine the correct SAR values.
+Ideally,
+you should cross-reference the results
+of as many of these methods
+as possible
+to derive the most accurate SAR values.
+
+!!! example "SAR heuristic methods"
+
+    === "Faded columns"
+
+        !!! warning "NTSC Active Area Standards"
+            NTSC has two common active area standards that are very similar:
+
+            - 710.85x486 (typically from analog transfers)
+            - 704x480 (more common in digital sources)
+
+            While the faded columns method below can help identify the active area,
+            additional analysis may be needed to definitively determine which standard is being used.
+
+        One way to determine the active area
+        is by looking for faded columns
+        on the edges of the frame.
+        These dark borders are intentionally added
+        to mark the boundaries of the active picture area.
+
+        ![Example frame from Nurse Witch Komugi Magikarte](./img/dvd-remux/sar/faded-columns-method.png)
```

Comment:
Here I'd explain very explicitly that the linked frame is a 720x480 raw frame of the DVD, just like it'd come out of the decoder, displayed with square pixels (and hence not how a CRT video player would display it).
These are exactly the things that confused me the most when I was learning - seeing guides on how to do certain conversions and not being sure what the example images signify and which steps in the process they come from.